packages/blockchain-info-components/src/Form/NumberInput.js

```diff
@@ -1,27 +1,31 @@
 import React from 'react'
+import PropTypes from 'prop-types'
 import styled from 'styled-components'
 
 const BaseNumberInput = styled.input.attrs({
   type: 'number'
 })`
   display: block;
   width: 100%;
-  height: 40px;
-  min-height: 40px;
+  height: ${props => props.height};
+  min-height: ${props => props.minHeight ? props.minHeight : '40px'};
   padding: 6px 12px;
   box-sizing: border-box;
   font-size: 14px;
   font-weight: 300;
   color: ${props => props.theme['gray-5']};
-  background-color: ${props => props.theme['white']};
+  background-color: ${props => props.disabled ? props.theme['gray-1'] : props.theme['white']};
   font-family: 'Montserrat', Helvetica, sans-serif;
   background-image: none;
   outline-width: 0;
   user-select: text;
   border: 1px solid  ${props => props.theme[props.borderColor]};
+  cursor: ${props => props.disabled ? 'not-allowed' : 'initial'};
```

Comment:
Why not just use the browsers default disabled functionality by setting the input to `disabled="true"`

Reply:
Actually, I just reused what was already created on TextInput. I can't remember the reason why I did not use the html attribute indeed, will investigate

Reply:
Ok well I removed it. Now the input is disabled but just show the default cursor | when you are on hover.